docs/service.md

```
@@ -65,6 +65,36 @@ Use `http://0.0.0.0:8080` to listen on all interfaces — but read the warning b
+    "RateLimit": "500/1h",
+    "RateLimitReplyCode": 451
+  }
+}
+```
+
+`RateLimit` is `<count>/<window>`, where the window is a number followed by `s`, `m`, or `h` —
+for example `500/1h`, `5/10m`, or `100/30s`. Use `*` (the default) for no limit.
+
+Once the limit is reached, Papercut rejects at `MAIL FROM` until the window resets:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)